### PR TITLE
Update system_advanced_network.php

### PR DESCRIPTION
**Pull request by marjohn56**
Add EN DUID Generation using FreeBSD PEN for now.

### CODE REVIEW

**Comment by marjohn56** [comment]
@fichtner - Have you requested a PEN?

**Comment by fichtner** [comment]
@marjohn56 it's 53869

**Comment by marjohn56** [comment]
Is done! I think. ;)

File: src/www/system_advanced_network.php [hunk 1]
```diff
@@ -86,6 +86,16 @@ function generate_new_duid($duid_type)
                 }
             }
             break;
+        case '4': //EN - USE FreeBSD PEN for now
```
Comment (fichtner) [inline src/www/system_advanced_network.php]: I don't think this is true anymore :)

File: src/www/system_advanced_network.php [hunk 2]
```diff
@@ -86,6 +86,16 @@ function generate_new_duid($duid_type)
...
+            $type = "\x00\x02\x0D\x02\x06\x0D".openssl_random_pseudo_bytes(8);
```
Comment (fichtner) [inline src/www/system_advanced_network.php]: shouldn't this be 00:02:00:00:D2:6D ?

**Comment by marjohn56** [comment]
It should, and that's what happens when you rush things in a morning hurrying to get out of the door!

Slapped hands..

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**